client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/style.scss

```diff
@@ -243,6 +242,11 @@
 	.launchpad__checklist-item-checkmark {
 		fill: var(--studio-blue-50);
 	}
+
+	// change badge color on hover (Choose a Plan step)
+	.badge {
+		background: var(--studio-blue-10);
+	}
```

Comment:
Noting this css specificity may not be ideal for this style. It looks like its original placement wasn't great either. But it has good behavior in the current state.

Before - it looks like this badge color was set up to change color on any task when hovered. Thats not ideal because when we have a task with a badge that is complete / inert this will still change color on hover. Currently we don't have any like that, but in the future this style will bleed to extra places where unexpected.

Now in this PR - we are making this specific to tasks with `keep-active`. I think this will neglect to take into consideration tasks that are active, incomplete, and not meant to keep-active after completion? - so this style may not take effect on some places where expected in the future.

Reply:
@Addison-Stavlo Great catch! I've updated the PR to include the hover and focus badge styles for pending tasks. I've set up a fake badge on a pending task for the purpose of testing:

https://user-images.githubusercontent.com/20927667/192341754-7d131248-13e6-47b2-9f62-80ba0d506151.mov